src/apps/frontend/pages/authentication/login/login-form.tsx

```diff
@@ -0,0 +1,85 @@
+import { Block } from 'baseui/block';
+import { KIND } from 'baseui/button';
+import { FormControl } from 'baseui/form-control';
+import { Input } from 'baseui/input';
+import { HeadingMedium } from 'baseui/typography';
+import React from 'react';
+
+import {
+  Button,
+} from '../../../components';
+
+import useLoginForm from './login-form.hook';
+
+interface LoginFormProps {
+  onSuccess: () => void;
+  onError: (error: string) => void;
+}
+
+const LoginForm: React.FC<LoginFormProps> = ({ onError, onSuccess }) => {
```

Comment:
I am not sure, can you confirm - 

When user submitted the form - the form controls are disabled, including login button as button also shows loading icon?

Reply:
Form controls are not disabled only login button is disabled with loading icon.

Reply:
We should disable form controls as well. When the login is in process we do not want user to edit their information.

Reply:
Done